lib/notification/adapter/mattermost.js

```diff
@@ -0,0 +1,59 @@
+const { markdown2Html } = require('../../services/markdown');
+const { getJob } = require('../../services/storage/jobStorage');
+const axios = require('axios');
+
+/**
+ * sends new listings to mattermost
+ * @param serviceName e.g immowelt
+ * @param newListings an array with newly found listings
+ * @param notificationConfig config of this notification adapter
+ * @param jobKey name of the current job that is being executed
+ * @returns {Promise<Void> | void}
+ */
+exports.send = ({ serviceName, newListings, notificationConfig, jobKey }) => {
```

Comment:
As far as I understood, this adapter sends messages per chat? If so, it is useful to use the telegram adapter, which shortens the results? Mattermost should not have the restriction of 30 chars no?

Reply:
Yes, thats true. I used it for better mobile view, but works without in the same way.  Changed it.